### ISSUE TITLE
Unable to create a MUC, IllegalArgumentException

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#325]
```

What steps will reproduce the problem?
1. Start Gtalk, connect to server
2. Start chat, attempt to create a chat room (MUC)

What is the expected output?
MUC opened, with user chat


What do you see instead?
GTalk SMS replies in XMPP client (pidgin) with:
ACTION_SMS_RECEIVED - Error writing to MUC: 
  -- caused by: java.lang.IllegalArgumentException: Field not found for the specified variable name.


Additional informations:
Phone model: Motorola Droid 4
```

Original issue reported on code.google.com by `dynamite...@gmail.com` on 9 May 2013 at 11:03

Attachments:
- [gtalk.log](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-325/comment-0/gtalk.log)

**Comment by GoogleCodeExporter** [comment]
```
I have tested this out, and it seems the server (conference.jabber.org) does 
not send the "muc#roomconfig_passwordprotectedroom" field in the configuration 
form. If you do not attempt to set this flag, the room is created successfully. 

One solution is to remove line 296 of XmppMuc.java, which sets the 
password-required flag. Another would be to check for the presence of the field 
and ignore it if the server does not require it (setting the password should be 
enough to say that the room shall be password protected). A third would be to 
always add the field if it is not requested by the server. 
```

Original comment by `dynamite...@gmail.com` on 11 Aug 2013 at 9:16

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 272a68c44c0d.
```

Original comment by `yako...@gmail.com` on 16 Sep 2013 at 10:14
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 44bb1821114f.
```

Original comment by `yako...@gmail.com` on 21 Sep 2013 at 1:29

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4.4.1
```

Original comment by `Florent....@gmail.com` on 28 Sep 2013 at 8:11
- Changed state: **Deployed**
